spec/grape_entity/entity_spec.rb

```diff
@@ -477,11 +477,20 @@ class Parent < Person
           expect(child_class.represent({ name: 'bar' }, serializable: true)).to eq(email: nil, name: 'foo')
         end
 
-        it 'overrides parent class exposure' do
+        it 'not overrides exposure by default' do
```

Comment:
did we have an example which overrides the exposure?

Reply:
Yep, I've used the spec I've written before https://github.com/ruby-grape/grape-entity/pull/297/files/0e86ebb5302ed9d2d4effc4d92eda7940abd13e1#diff-2f09e9dc67c5575f0172b3d0defe0e1bR489

Reply:
ok … thanks for clearing